conmon/conmon.c

```diff
@@ -884,7 +884,8 @@ static void write_sync_fd(int sync_pipe_fd, int res, const char *message)
 static char *setup_console_socket(void)
 {
 	struct sockaddr_un addr = {0};
-	char csname[PATH_MAX] = "/tmp/conmon-term.XXXXXXXX";
+	_cleanup_free_ const char *tmpdir = g_get_tmp_dir();
+	_cleanup_free_ char *csname = g_build_filename(tmpdir, "conmon-term.XXXXXX", NULL);
```

Comment:
The slash is added by `g_build_filename` which underneath uses `g_build_path (G_DIR_SEPARATOR_S, first_element, ....).` using `/` as `G_DIR_SEPARATOR` by default.